test/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 
 services:
   aperturedb:
-    image: aperturedata/aperturedb
+    image: aperturedata/aperturedb:vdevelop
```

Comment:
Why the change in image?

Reply:
Same as above

Reply:
cool.

Reply:
[v0.15.16](https://github.com/aperture-data/athena/compare/v0.15.15...v0.15.16) is ready to go!

Reply:
I think we can resolve this